<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyBrowser</class>
 <widget class="QWidget" name="MyBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="backButton">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="forwardButton">
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="homeButton">
       <property name="text">
        <string>H</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWebView" name="webView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="toolPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>185</width>
         <height>453</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tools</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="savePageButton">
          <property name="text">
           <string>Save page content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="viewDBButton">
          <property name="text">
           <string>View DB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="crawlButton">
          <property name="text">
           <string>Crawl</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="linksPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>185</width>
         <height>453</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Links</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="linksView">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imagesPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>185</width>
         <height>453</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="imagesView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="inspectorPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>185</width>
         <height>453</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Inspector</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWebInspector" name="webInspector" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>QWebInspector</class>
   <extends>QWidget</extends>
   <header location="global">QWebInspector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>webView</tabstop>
  <tabstop>backButton</tabstop>
  <tabstop>forwardButton</tabstop>
  <tabstop>homeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>webView</sender>
   <signal>loadProgress(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>644</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>24</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>titleChanged(QString)</signal>
   <receiver>MyBrowser</receiver>
   <slot>setWindowTitle(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
